data/wanderer/wanderers start.txt

```diff
@@ -179,67 +183,388 @@ mission "Wanderers: Hai Diplomat"
+				accept
+
+
+event "A Hai professor is available to translate"
+
+
+
+mission "Wanderers: Sayari Available After Hai Reveal"
+	landing
+	name "Wanderers: Pick Up Sayari"
+	description "The Hai government have chosen an ambassador for the aliens to the north of Hai space. Pick them up at <destination>."
+	source
+		not planet Hai-home
+		not attributes uninhabited
+		government Hai Republic Syndicate "Free Worlds" Neutral Independent
+	destination Hai-home
+	to offer
+		has "Wanderers: Diplomacy Delayed: offered"
+		not "event: Wanderers before Hai Reveal"
+		has "event: Sayari available after Hai Reveal"
+	on offer
+		fail "Wanderers: Diplomacy Delayed"
+		fail "Wanderers: Professor Available"
+		fail "Wanderers: Meet Professor"
+		conversation
+			`Upon landing, you receive an official communique from the Hai government. "A diplomat is available for reassignment to first contact with the avian species you met to the north. Please meet her as soon as possible at <planet>.`
+				accept
+
+
+mission "Wanderers: Professor Available"
+	landing
+	name "Wanderers: Professor Andelena"
+	description "Fly to <planet> to pick up a professor and grad student who are willing to translate for you."
+	source
+		not planet Hai-home
+		not planet Allhome
+		not attributes uninhabited
+		government Hai Republic Syndicate "Free Worlds" Neutral Independent
+	destination Allhome
+	to offer
+		has "event: A Hai professor is available to translate"
+		not "Wanderers: Diplomacy: done"
+	on offer
+		log `Professor Andelena in the Department of Skiourology at Mongongo University has sent me a message asking for transport services on her expedition. This may be an opportunity to get an interpreter for the aliens to the north.`
+		conversation
+			`Your ship's terminal bleats out a pinging noise, accompanied by a message from a university on <planet>. "Professor Andelena in the Department of Skiourology at Mongongo University awaits your transport services on her expedition." This may be the interpreter you've been waiting for.`
+				accept
+
+
+
+mission "Wanderers: Meet Professor"
+	invisible
+	landing
+	name "Wanderers: Professor Andelena"
+	source Allhome
+	to offer
+		has "Wanderers: Professor Available: done"
+		not "Wanderers: Sayari Available After Hai Reveal: offered"
+	to complete
+		has "Wanderers: Diplomacy (Before Hai Reveal): offered"
+	on offer
+		log "Minor People" "Professor Andelena" `Professor Maindell Stormchild Andelena, IV speaks with Shroud cloud city accent and is a daughter of the rich Andelena family. Her wealthy meanderings have led her to a university on Allhome where she studies Skiorology, the Hai-studying equivalent of Anthropology.`
```

Comment:
```suggestion
		log "Minor People" "Professor Andelena" `Professor Maindell Stormchild Andelena IV speaks with a Shroud cloud city accent and is a daughter of the rich Andelena family. Her wealthy meanderings have led her to a university on Allhome where she studies Skiorology, the Hai-studying equivalent of Anthropology.`
```

Reply:
I prefer it without the comma, so I'll accept the suggestion. We shall see if someone asks to change it back.